### DISCUSSION TITLE
Node version is fixed

### DISCUSSION THREAD

**Discussion by assertnotnull** [remix-run/remix#5407] [Q&A]
### What version of Remix are you using?

1.12.0

### Are all your remix dependencies & dev-dependencies using the same version?

- [X] Yes

### Steps to Reproduce

In a new project:
1. Install volta.sh
1. `volta pin node@18`
2. check `node -v` is v18
1. in a server file such `entry.server.ts` add : 
```
import { version } from "process";
console.log("Node version:", version);
```

### Expected Behavior

Should print `node version v18`

### Actual Behavior

It prints `node version v16`

**Comment by machour** [comment]
Hey,

You're failing to mention if you have started the server **after** switching to node 18 or not.
Please include a full log

**Reply by assertnotnull** [reply]
Here's a recording of it
https://asciinema.org/a/K2UtmriPVHUWzd2EhDcnspCes
